test/subreddit/lib_subreddit_about_moderators.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Did you write this manually? If so, the test won't work properly since the test harness basically records and replays the network requests (authentication requests aren't recorded). Is this just because you weren't able to authenticate properly?

Reply:
No, this is plain copy from https://www.reddit.com/r/dartlang/about/moderators.json